src/scripts/blog.js

```diff
@@ -0,0 +1,14 @@
+import { createArticle } from './createArticle.js';
+import { setCorrectViewType } from './viewType.js';
+
+function displayBlogPosts(data) {
+  const blogData = data.slice(0, 6);
+  for (const post of blogData) {
+    createArticle(post.title.rendered, post.jetpack_featured_media_url, post.excerpt.rendered, post.date_gmt + 'Z');
```

Comment:
вот кстати если вспомнить лабу, из 2ого модуля, про деструктуризацию
можно передавать объект и получатся именованные параметры
```suggestion
    createArticle({
      title: post.title.rendered,
      media_url:  post.jetpack_featured_media_url,
      excerpt:  post.excerpt.rendered,
      date: post.date_gmt + 'Z'
    });
```

и так гораздо удобней и наглядней что именно передается в функцию,

исправлять не нужно. 
это просто пример